phenol-cli/src/main/java/org/monarchinitiative/phenol/cli/demo/PairwisePhenotypicSimilarityCalculator.java

```diff
@@ -326,8 +326,9 @@ public void run() {
     this.hpo = OntologyLoader.loadOntology(new File(pathHpObo));
     System.out.println("[INFO] DONE: Loading HPO");
 
-    List<DiseaseDatabase> databases = ImmutableList.of(OMIM); // restrict ourselves to OMIM entries
-    this.diseaseMap = HpoDiseaseAnnotationParser.loadDiseaseMap(this.pathPhenotypeHpoa, hpo,databases);
+    Set<DiseaseDatabase> databases = new HashSet<>(); // restrict ourselves to OMIM entries
```

Comment:
Said it before, but this should be `Set.of(OMIM)`, but I don't see what's wrong with a List as input, you can ensure no duplicates internally.

Reply:
My reasoning is described above.

Reply:
Sure, but why not `Set.of(OMIM)` instead of the two liner?

Reply:
We're still Java 8 here. :/